device/dell/x86_64-dellemc_z9332f_d1508-r0/platform.json

```diff
@@ -611,5 +611,340 @@
 
         ]
     },
-    "interfaces": {}
+    "interfaces": {
+        "Ethernet0": {
+            "index": "1,1,1,1,1,1,1,1",
+             "lanes": "33,34,35,36,37,38,39,40",
+             "breakout_modes": {
+                "1x400G": ["etp1"],
+                "2x200G[100G,40G]": ["etp1a", "etp1b"],
```

Comment:
can we add 2x100G also ?

Reply:
Orchagent crashes while trying to configure 2x100G directly.
        Nov 22 04:34:53.863462 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_PORT_ID: oid:0x10000000005b3
        Nov 22 04:34:53.863462 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_TX_FIR_PRE1: 4:-28,-28,-28,-28
        Nov 22 04:34:53.863491 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_TX_FIR_PRE2: 4:2,4,0,4
        Nov 22 04:34:53.863491 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_TX_FIR_MAIN: 4:136,136,140,136
        Nov 22 04:34:53.863491 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_TX_FIR_POST1: 4:0,0,0,0
        Nov 22 04:34:53.863501 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_TX_FIR_POST2: 4:0,0,0,0
        Nov 22 04:34:53.863516 sonic ERR syncd#syncd: :- processQuadEvent: attr: SAI_PORT_SERDES_ATTR_TX_FIR_POST3: 4:0,0,0,0
        Nov 22 04:34:53.863850 sonic ERR swss#orchagent: :- create: create status: SAI_STATUS_INVALID_ATTR_VALUE_0
        Nov 22 04:34:53.863850 sonic ERR swss#orchagent: :- setPortSerdesAttribute: Failed to create port serdes for port 0x10000000005b3
        Nov 22 04:34:53.863850 sonic ERR swss#orchagent: :- handleSaiCreateStatus: Encountered failure in create operation, exiting orchagent, SAI API: SAI_API_PORT, status: SAI_STATUS_INVALID_ATTR_VALUE_0